apps/avatax/package.json

```diff
@@ -45,8 +45,8 @@
     "@saleor/react-hook-form-macaw": "workspace:*",
     "@saleor/sentry-utils": "workspace:*",
     "@saleor/webhook-utils": "workspace:*",
-    "@sentry/cli": "2.31.2",
-    "@sentry/nextjs": "8.0.0",
+    "@sentry/cli": "../../node_modules/@sentry/cli",
+    "@sentry/nextjs": "../../node_modules/@sentry/nextjs",
```

Comment:
is it intentional?

Reply:
Yes - it help us with keeping our deps in sync between different repos

Reply:
and we reduce risk of multiple sentry versions included, eg
1. App X uses Sentry
2. App Y uses Sentry - can be different version
3. Package/lib Z uses Sentry - has it in devDependences, or dependencies, or peer

It happened that module resolution went crazy, especially for packages that heavily affect global scope, use proxies, singleton etc